<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartForm</class>
 <widget class="QWidget" name="StartForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="downloadImageButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Загрузить фото</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>81</y>
     <width>611</width>
     <height>391</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="binarizationButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>бинаризовать</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="binarizationSpinBox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>30</y>
     <width>151</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maximum">
    <number>256</number>
   </property>
   <property name="value">
    <number>245</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="tresholdSpinBox">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>30</y>
     <width>141</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimum">
    <number>50</number>
   </property>
   <property name="maximum">
    <number>1000000</number>
   </property>
  </widget>
  <widget class="QPushButton" name="fileterButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>30</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Отфильтравать</string>
   </property>
  </widget>
  <widget class="QPushButton" name="allAcinusButton">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>80</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>все ацинусы</string>
   </property>
  </widget>
  <widget class="QPushButton" name="healthAcinusButton">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>110</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Здоровые</string>
   </property>
  </widget>
  <widget class="QPushButton" name="threadGlissonButton">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>140</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Глиссон 3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="forthGlissonButton">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>170</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>глиссон 4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="findAnalogInDb">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>470</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Найти похожие в базе</string>
   </property>
  </widget>
  <widget class="QLabel" name="resultLable">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>210</y>
     <width>241</width>
     <height>221</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>450</y>
     <width>99</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>сохранить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="gistButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>470</y>
     <width>191</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>посмотреть гистограмму</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
